src/py42/modules/securitydata.py

```diff
@@ -52,6 +59,39 @@ def get_plan_security_events(
         min_timestamp=None,
         max_timestamp=None,
     ):
+        """Gets legacy security events.
+
+        Args:
+            plan_storage_info (:class:`py42.sdk.modules.securitydata.PlanStorageInfo`):
+                Information about storage nodes for a plan to get security events for.
+            cursor (str, optional): A cursor position for only getting events you did not
+                previously get. Defaults to None.
+            include_files (bool, optional): Whether to include the files related to the security
+                events. Defaults to None.
+            event_types: (str, optional): A comma-separated list of event types to filter by.
+
+                    Available options are:
+                        - 'DEVICE_APPEARED'
+                        - 'DEVICE_DISAPPEARED'
+                        - 'DEVICE_FILE_ACTIVITY'
+                        - 'PERSONAL_CLOUD_FILE_ACTIVITY'
+                        - 'RESTORE_JOB'
+                        - 'RESTORE_FILE'
+                        - 'FILE_OPENED'
+                        - 'RULE_MATCH'
+                        - 'DEVICE_SCAN_RESULT'
+                        - 'PERSONAL_CLOUD_SCAN_RESULT'
+
+                    Defaults to None.
+            min_timestamp (float, optional): A POSIX timestamp to filter out events that did not
```

Comment:
Looks like since the endpoint itself takes a date format string instead of a timestamp, we're just requiring a timestamp for consistency here. So they'd all be seconds as that's what `datetime.utcfromtimestamp()` takes.